torch/csrc/jit/import.cpp

```diff
@@ -80,16 +80,16 @@ class ScriptModuleDeserializer final {
 };
 
 ScriptModuleDeserializer::ScriptModuleDeserializer(const std::string& filename)
-    : reader_(filename.c_str()) {
+    : reader_(filename.c_str()), attribute_table_(c10::impl::make_generic_list()) {
```

Comment:
I don't see attribute_table_ at all, but it shouldn't be GenericList

Reply:
fixed, this change was based on an old version